plugins/premium/powerpaste.md

```diff
@@ -95,48 +95,105 @@ tinymce.init({
 
 This option controls how content pasted from Microsoft Word is filtered.
 
-**Type:** `String`
+**Type:** `String` or `function`
 
 **Default value:** `prompt`
 
-The supported values are:
+The supported `string` values are:
```

Comment:
This uses a different casing to the above types. We have to use the types that match moxiedoc in the above, so not sure if this should also be `String`? Maybe it shouldn't be in backticks and instead just say something like this?
```suggestion
The supported string-based values are:
```